sounder/sounder.py

```diff
@@ -32,11 +32,11 @@ def get_reserved_sub_words():
             "what", "where", "which", "how", "when", "who",
             "is", "are", "makes", "made", "make", "did", "do",
             "to", "the", "of", "from", "against", "and", "or",
-            "you", "me", "we", "us", "your", "my", "mine", 'yours',
+            "you", "me", "we", "us", "your", "my", "mine", "yours",
             "could", "would", "may", "might", "let", "possibly",
-            'tell', "give", "told", "gave", "know", "knew",
-            'a', 'am', 'an', 'i', 'like', 'has', 'have', 'need',
-            'will', 'be', "this", 'that', "for"
+            "tell", "give", "told", "gave", "know", "knew",
```

Comment:
so proud

Reply:
revolutionary! This warrants a scholarship!

Reply:
WTF, why is my alt GitHub account @i338395 in the participants? :o

Reply:
Where is Linux foundation when you need them